src/tools/rbd_mirror/ImageReplayer.cc

```diff
@@ -456,6 +502,28 @@ void ImageReplayer<I>::handle_init_remote_journaler(int r) {
     return;
   }
 
+  {
```

Comment:
Should this be moved to ```handle_bootstrap```? That's the first time that the local image (and associate journal) is available to check.